src/diagnostics.md

```diff
@@ -594,6 +594,20 @@ declare_lint! {
 This makes the `ANONYMOUS_PARAMETERS` lint allow-by-default in the 2015 edition
 but warn-by-default in the 2018 edition.
 
+### Feature-gated lints
+
+Lints belonging to a feature should only be usable if the feature is enabled in the
+crate. To support this, lint declarations can contain a feature gate like this:
```

Comment:
```suggestion
crate. To support this, lint declarations can contain a feature gate like so:
```

Reply:
`this` is one of the words I use way to often, nice improvement :upside_down_face: